util/kvcache/simple_lru.go

```diff
@@ -169,3 +169,17 @@ func (l *SimpleLRUCache) Keys() []Key {
 	}
 	return keys
 }
+
+// SetCapacity sets capacity of the cache.
+func (l *SimpleLRUCache) SetCapacity(capacity uint) {
+	if capacity < 1 {
+		panic("capacity of LRU Cache should be at least 1.")
```

Comment:
I think the better way to handle the invalid capacity is produce an error instead of panicking.